frontend/src/components/GoalForm/index.js

```diff
@@ -779,10 +779,10 @@ export default function GoalForm({
         ids,
         prompts,
       });
-
       const goals = [
         ...createdGoals.reduce((acc, goal) => {
           const g = grantsToGoals({
+            // ...goal,
```

Comment:
remove comment?

Reply:
Did you push up your latest stuff? I'm staring at it in Github right now on a fresh browser tab

Reply:
done.